src/services/db/GitFileSystemService.ts

```diff
@@ -476,12 +551,46 @@ export default class GitFileSystemService {
       })
   }
 
+  fastForwardMaster(
+    repoName: string
+  ): ResultAsync<boolean, GitFileSystemError> {
+    const efsVolPath = this.getEfsVolPathFromBranch("master")
+    return this.isValidGitRepo(repoName, "master").andThen((isValid) => {
+      if (!isValid) {
+        return errAsync(
+          new GitFileSystemError(
+            `Folder "${repoName}" for EFS vol path: "${efsVolPath}" is not a valid Git repo`
+          )
+        )
+      }
+      return this.createLocalTrackingBranchIfNotExists(repoName, "master")
+        .andThen(() =>
+          ResultAsync.fromPromise(
+            this.git
+              .cwd({ path: `${efsVolPath}/${repoName}`, root: false })
```

Comment:
lets use fs.path for this rather than constructing this by ourselves

Reply:
[6931527](https://github.com/isomerpages/isomercms-backend/pull/1172/commits/6931527c71f805ec5b316922a88d17ec02ab81e4)